dockerfiles/node-bench-regression-guard/run-tests.rb

```diff
@@ -0,0 +1,111 @@
+require 'faker'
+require 'fileutils'
+require 'json'
+require 'securerandom'
+
+require 'minitest/autorun'
+
+def in_tmp_dir
+  tmpdir = "/tmp/nbrg-tests-#{SecureRandom.hex[0..10]}"
```

Comment:
[Tempfile](https://ruby-doc.org/stdlib-2.7.0/libdoc/tempfile/rdoc/Tempfile.html#method-c-new) is actually part of the stdlib, so you could use this.

Reply:
Found related method [`Dir::mktmpdir`](https://ruby-doc.org/stdlib-2.7.2/libdoc/tmpdir/rdoc/Dir.html)! Thanks.